task3/update/index.html

```diff
@@ -0,0 +1,336 @@
+<!DOCTYPE html>
```

Comment:
First of all, please leave just one version, you can use the same branch and PR just add new commits.